pkg/rtc/room.go

```diff
@@ -1099,18 +1099,20 @@ func (r *Room) subscribeToExistingTracks(p types.LocalParticipant) {
 
 // broadcast an update about participant p
 func (r *Room) broadcastParticipantState(p types.LocalParticipant, opts broadcastOptions) {
+	pi := p.ToProto()
```

Comment:
should use `ToProtoWithVersion` here to make sure the proto buf and version are synced